packages/react-storage/src/components/StorageBrowser/context/elements/defaults.tsx

```diff
@@ -15,9 +15,13 @@ import {
   View as _View,
   ViewProps,
 } from '@aws-amplify/ui-react';
+import { ButtonElementProps, LabelElement, TextElement } from './definitions';
 
-const Button = React.forwardRef<HTMLButtonElement>(function Button(props, ref) {
-  const { disabled, variant } = props as any;
+const Button = React.forwardRef<HTMLButtonElement>(function Button(
```

Comment:
Can provide `ButtonElementProps` as the second positional generic of `React.forwardRef` and remove the typing on lines 21 and 22 (same feedback for `Label`)